app/src/main/java/org/listenbrainz/android/application/App.kt

```diff
@@ -55,8 +53,8 @@ class App : Application(), Configuration.Provider {
         MainScope().launch {
             if(
                 appPreferences.isNotificationServiceAllowed &&
-                appPreferences.getLbAccessToken().isNotEmpty() &&
-                withContext(Dispatchers.IO) { appPreferences.submitListens }
+                appPreferences.lbAccessToken.get().isNotEmpty() &&
+                appPreferences.isScrobblingAllowed.get()
```

Comment:
We decided on IRC to not use the term Scrobble. I think this is a good time to fix this.

Reply:
Alright

Reply:
@07jasjeet have we fixed this?

Reply:
Please fix this @07jasjeet

Reply:
Its already fixed, this is an older commit.

Reply:
Cool